extensions/azurecore/src/azureResource/services/terminalService.ts

```diff
@@ -31,9 +60,15 @@ export class AzureTerminalService implements IAzureTerminalService {
 		};
 
 		const metadata = account.properties.providerSettings;
-
 		const userSettingsUri = this.getConsoleUserSettingsUri(metadata.settings.armResource.endpoint);
-		const userSettingsResult = await axios.get(userSettingsUri, settings);
+
+		let userSettingsResult: AxiosResponse<any>;
+		try {
+			userSettingsResult = await axios.get(userSettingsUri, settings);
+		} catch  {
+			await handleNeverUsed();
```

Comment:
Let's also log the error since it's possible something else went wrong and we'll want to see what that is